src/components/Status/LeadingOption.tsx

```diff
@@ -7,53 +7,53 @@ import { ProposalStatus } from '../../entities/Proposal/types'
 
 import './LeadingOption.css'
 
-export type LeadingOptionProps = React.HTMLAttributes<HTMLDivElement> & {
+const FINISHED_STATUSES = [
+  ProposalStatus.Passed,
+  ProposalStatus.Rejected,
+  ProposalStatus.Finished,
+  ProposalStatus.OutOfBudget,
+  ProposalStatus.Enacted,
+]
+
+interface Props {
   leadingOption: string | null
   userChoice: string | null
   metVP: boolean
   status: ProposalStatus
 }
 
-export default React.memo(function LeadingOption({ status, leadingOption, metVP, userChoice }: LeadingOptionProps) {
+export default React.memo(function LeadingOption({ status, leadingOption, metVP, userChoice }: Props) {
   const t = useFormatMessage()
   const [user] = useAuthContext()
 
   const showLeadingOption = !user || !!userChoice
 
   const proposalFinished = useMemo(() => {
-    return [
-      ProposalStatus.Passed,
-      ProposalStatus.Rejected,
-      ProposalStatus.Finished,
-      ProposalStatus.OutOfBudget,
-      ProposalStatus.Enacted,
-    ].includes(status)
+    return FINISHED_STATUSES.includes(status)
```

Comment:
I wouldn't move this to a helper if we are not reusing it anywhere else. But in this case, as it can make the platform more "understandable" we could do that. What could be a good name for this? As calling it "finished" doesn't seem right having a "Finished" status.